vendor/init.bat

```diff
@@ -200,14 +201,14 @@ if defined GIT_INSTALL_ROOT (
 
 :: check if git is in path...
 for /F "delims=" %%F in ('where git.exe 2^>nul') do (
-    :: get the absolute path to the user provided git binary
-    %lib_git% is_git_shim "%%~dpF"
-    %lib_git% get_user_git_version
-    %lib_git% compare_git_versions
-)
+    if not defined GIT_INSTALL_ROOT (
+        :: get the absolute path to the user provided git binary
+        %lib_git% is_git_shim "%%~dpF"
+        %lib_git% get_user_git_version
+        %lib_git% compare_git_versions
 
-if defined GIT_INSTALL_ROOT (
-    goto :FOUND_GIT
+        goto :FOUND_GIT
+    )
```

Comment:
This makes the script always go to `FOUND_GIT` which does not seem intended, e.g. `%lib_git% compare_git_versions` does not define `GIT_INSTALL_ROOT` when the checked git is older than the vendored git.

So this should probably be checked at the end of each iteration instead:
```suggestion
    :: get the absolute path to the user provided git binary
    %lib_git% is_git_shim "%%~dpF"
    %lib_git% get_user_git_version
    %lib_git% compare_git_versions

    if defined GIT_INSTALL_ROOT (
        goto :FOUND_GIT
    )
```